vagrant/ubuntu-xenial/Vagrantfile

```diff
@@ -9,9 +9,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.box = "ubuntu/xenial64"
   config.ssh.insert_key = false
-  config.vm.hostname = "demo.vagrant1.ala.org.au"
+  config.vm.hostname = "vagrant1.ala.org.au"
```

Comment:
What is the background to making this change?

Reply:
I have a vague memory, somewhere we use vagant1, instead of demo.vagrant1